.eslintrc

```diff
@@ -1,6 +1,124 @@
 {
```

Comment:
@geek @rjrodger This is temporary until we do what needs to be done with Lab, it means that linting can be done while we wait and still remove the standard modules.